backend/__tests__/user.test.js

```diff
@@ -197,4 +199,33 @@ describe("all profile updating", () => {
       .set("Content-Type", "application/json");
     expect(response.status).toEqual(403);
   });
+
+  it("should update a user's bio", async () => {
+    const newBio = "This is the updated bio of Tommy Tester.";
+    const response = await supertest(app)
+      .post("/api/users/updateUser")
+      .send({
+        id: "197bdce9-f60b-4529-b781-a78fb54d7c51",
+        bio: newBio
+      })
+      .set("Authorization", `Bearer ${token}`)
+      .set("Content-Type", "application/json")
+      .set("Accept", "application/json");
+
+    expect(response.status).toEqual(200);
+    expect(response.body.bio).toEqual(newBio);
+  });
+
+  it("should not allow a user to update someone else's bio", async () => {
+    const newBio = "This is a user's attempt to update another user's bio.";
+    const response = await supertest(app)
+      .post("/api/users/updateUser")
+      .set({ id: "8b414dd1-f7cb-482c-9f4a-3cfdb998f948", bio: newBio })
```

Comment:
Typo here? Should be `send()` for the request body
```suggestion
      .send({ id: "8b414dd1-f7cb-482c-9f4a-3cfdb998f948", bio: newBio })
```